test/functional/Java14andUp/src/org/openj9/test/jep358/NPEMessageTests.java

```diff
@@ -26,26 +26,26 @@
 
 import org.testng.Assert;
 
-import jdk.internal.org.objectweb.asm.ClassWriter;
-import jdk.internal.org.objectweb.asm.Label;
-import jdk.internal.org.objectweb.asm.MethodVisitor;
-import jdk.internal.org.objectweb.asm.Opcodes;
-import jdk.internal.org.objectweb.asm.Type;
+import org.objectweb.asm.ClassWriter;
+import org.objectweb.asm.Label;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.Type;
 
 import static java.lang.invoke.MethodHandles.lookup;
 
-import static jdk.internal.org.objectweb.asm.Opcodes.ACC_PUBLIC;
-import static jdk.internal.org.objectweb.asm.Opcodes.ACC_SUPER;
-import static jdk.internal.org.objectweb.asm.Opcodes.ACONST_NULL;
-import static jdk.internal.org.objectweb.asm.Opcodes.ALOAD;
-import static jdk.internal.org.objectweb.asm.Opcodes.ASTORE;
-import static jdk.internal.org.objectweb.asm.Opcodes.GETFIELD;
-import static jdk.internal.org.objectweb.asm.Opcodes.GETSTATIC;
-import static jdk.internal.org.objectweb.asm.Opcodes.ICONST_1;
-import static jdk.internal.org.objectweb.asm.Opcodes.INVOKESPECIAL;
-import static jdk.internal.org.objectweb.asm.Opcodes.INVOKEVIRTUAL;
-import static jdk.internal.org.objectweb.asm.Opcodes.IRETURN;
-import static jdk.internal.org.objectweb.asm.Opcodes.RETURN;
+import static org.objectweb.asm.Opcodes.ACC_PUBLIC;
```

Comment:
Can we keep the references to jdk internal ASM to avoid updating external asm.jar file later?
A workaround for `IllegalAccessError` could be something like `--add-exports java.base/jdk.internal.org.objectweb.asm=ALL-UNNAMED`.

Reply:
Hi @JasonFengJ9 , currently  the --add-exports has already exist in the [build.xml](https://github.com/eclipse/openj9/blob/67bb44416a1caafdbcebdff6a2463b44d82f13cd/test/functional/Java14andUp/build.xml#L81), but it still has `IllegalAccessError`, could you specify where we also need to add this? Thanks.

Reply:
> --add-exports has already exist in the build.xml

That's for compilation, it need to be set for runtime after JDK tights its access control.
The option can be set at https://github.com/eclipse/openj9/blob/67bb44416a1caafdbcebdff6a2463b44d82f13cd/test/functional/Java14andUp/playlist.xml#L83
`JEP358NPEMessageTests-noDebugInfo` needs similar change as well.

Reply:
The commit has been updated, and it passed on all platforms. Thanks a lot!